### DISCUSSION TITLE
Tuangkan ide atau saranmu disini

### DISCUSSION THREAD

**Discussion by IrvanSN** [semicolon-coder/DOC_Titikoma_Semicolon-FE#2] [Ideas]
Halo teman-teman semuanya.
Kalau ada ide atau saran yang bagus dan memungkinkan untuk diterapkan bisa di reply discussion ini yaa!

Terima kasih!

**Comment by iqbalmusyaffa21** [comment]
Mungkin bisa ditambahi fitur foto promosi di frontend nya biar menarik dan biar gak keliru meng input pesanan

**Reply by IrvanSN** [reply]
Okee, saran saya tampung dulu. Terima kasih!

**Comment by mharits-s** [comment]
![Screenshot_20220325-201022~2](https://user-images.githubusercontent.com/100942834/160126986-58045cfc-3a9f-4a5b-9e6a-cff0131ba77e.png)
Saya sarankan untuk logo utama pada bagian atas ini bisa direct ke home page

**Reply by IrvanSN** [reply]
Okee mas, saran saya tampung. Silahkan bikin todo di https://github.com/orgs/semicolon-coder/projects/1 terus klik titik 3, pilih Convert to issue ya mas. arahkan ke repository ini juga.

**Comment by mharits-s** [comment]
![Screenshot_20220325-201654~2](https://user-images.githubusercontent.com/100942834/160128150-1a8441bc-f890-436f-aa85-1cc1cca3c2b9.png)
Kemudian saya sarankan untuk menambah prefix "Rp" pada nominal harga

**Reply by IrvanSN** [reply]
Okee baik saran saya tampung. Silahkan bikin todo di Projects Board kemudian convert ke issues ya mas.
